### PR TITLE
[Improvements] GitHub templates

### PR DESCRIPTION
**Pull request by evertonlperes**
### Changes
1. Removed preloaded title tag for Bug and Features request (Jan's request)
2. Added K8s Version input

Preview
* [Bug](https://github.com/rancher-sandbox/rancher-desktop/blob/c386d5eacffdac62ce80b9c7edce097caa346e1f/.github/ISSUE_TEMPLATE/bug_report.yml)
* [Feature](https://github.com/rancher-sandbox/rancher-desktop/blob/c386d5eacffdac62ce80b9c7edce097caa346e1f/.github/ISSUE_TEMPLATE/feature_request.yml)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Bug Report
 description: Report Rancher Desktop issue
-title: "[Bug]- "
+title: " "
```
Comment (jandubois) [inline .github/ISSUE_TEMPLATE/bug_report.yml:3]: Can't this be the empty string? Would the new setting prefix every title with a space?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -10,6 +10,13 @@ body:
     placeholder: 0.6.1
   validations:
     required: true
+- type: input
+  attributes:
+    label: Rancher Desktop K8s Version
+    description: What version of Kubernetes are you using?
+    placeholder: 1.21.6
```
Comment (jandubois) [inline .github/ISSUE_TEMPLATE/bug_report.yml:17]: TO make it look like the other placeholders:

```suggestion
    placeholder: "e.g. 1.21.6"
```

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (evertonlperes) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Leaving it empty or without space throws an error:
Screen Shot 2021-11-17 at 12 39 07 PM
I think it must
Screen Shot 2021-11-17 at 12 43 50 PM

Best solution: remove the entire tag 😄

**Review by jandubois — APPROVED** [review]
Thanks, LGTM

**Pull request merged**